gobblin-compaction/src/main/java/org/apache/gobblin/compaction/verify/CompactionAuditCountVerifier.java

```diff
@@ -118,17 +131,21 @@ private static AuditCountClientFactory getClientFactory (State state) {
    * @return If verification is succeeded
    */
   public Result verify (FileSystemDataset dataset) {
+    if (!enabled) {
+      return new Result(true, "");
+    }
     if (auditCountClient == null) {
       log.debug("No audit count client specified, skipped");
       return new Result(true, "");
     }
 
-    CompactionPathParser.CompactionParserResult result = new CompactionPathParser(this.state).parse(dataset);
-    DateTime startTime = result.getTime();
-    DateTime endTime = startTime.plusHours(1);
+    CompactionPathParser.CompactionParserResult result = new CompactionPathParser(state).parse(dataset);
+    ZonedDateTime startTime = ZonedDateTime.ofInstant(Instant.ofEpochMilli(result.getTime().getMillis()), zone);
```

Comment:
I'm not quite sure if we really need to convert to ZoneDateTime first. Seems like current joda DateTime already supports plusMinutes,  plusHours, plusDays, plusMonths.

Reply:
Because `TimeIterator.inc` accepts `ZoneDateTime`. I implemented `TimeIterator` using java.time as joda time is kind of deprecated by java 8+. 

```
The standard date and time classes prior to Java SE 8 are poor. By tackling this problem head-on, Joda-Time became the de facto standard date and time library for Java prior to Java SE 8. Note that from Java SE 8 onwards, users are asked to migrate to java.time (JSR-310) - a core part of the JDK which replaces this project.
```